recipe/activate.sh

```diff
@@ -1,7 +1,31 @@
 #!/bin/bash
 
-if [[ -n "$CXX" ]]; then
-    export NVCC_PREPEND_FLAGS_BACKUP="${NVCC_PREPEND_FLAGS}"
-    export NVCC_PREPEND_FLAGS="${NVCC_PREPEND_FLAGS} -ccbin=${CXX}"
+export CFLAGS_BACKUP="${CFLAGS}"
```

Comment:
Do we need to add a deactivation script to restore the backups?

Reply:
Yep, it's in my todos above. Maybe it should be a checklist?